srv/components/misc_pkgs/mocks/cortx/_macros.sls

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# TODO: use path from pillars or any other configuration
```

Comment:
[Macros in Jinja](https://jinja.palletsprojects.com/en/2.11.x/templates/#macros) are like functions in programming langs: they allow (among other things) to follow DRY principle avoind copy-paste. Salt supports that as well.

That trick we [have used already](https://github.com/Seagate/cortx-prvsnr/blob/4c20179804329b26d7f0cc0cf6b346f4cf015b18/srv/components/misc_pkgs/swupdate/repo/install.sls#L18)

But mocks.md is not a good place since that is mostly for devs. I will create a separate doc.

Regarding the `_`: the same thing as for python, it is going to be imported by other SLS, and not called via salt command or be included to states.

Reply:
added one more [docs for saltstack](docs/saltstack.md)